src/main/flight/pid_init.c

```diff
@@ -104,11 +104,27 @@ static void tpaSpeedAdvancedInit(const pidProfile_t *pidProfile)
     const float c = -mass * (pidRuntime.tpaSpeed.twr + 1) * G_ACCELERATION;
 
     const float maxDiveSpeed = (-b + sqrtf(b*b - 4.0f * a * c)) / (2.0f * a);
-
     pidRuntime.tpaSpeed.maxSpeed = MAX(maxFallSpeed, maxDiveSpeed);
+
     UNUSED(pidProfile);
 }
 
+static void aerodynamicsInit(const pidProfile_t *pidProfile)
+{
+    pidRuntime.planeAerodynProperty.mode = (aerodynamicsMode_e)pidProfile->ad_mode;
```

Comment:
(maybe save some typing)
```suggestion
    aerodynamicsProperty_t *adp = &pidRuntime.planeAerodynProperty;
    adp->mode = (aerodynamicsMode_e)pidProfile->ad_mode;
```